src/snovault/elasticsearch/__init__.py

```diff
@@ -16,12 +18,79 @@
 )
 import json
 import sys
+
+from redis import StrictRedis
+
+
 PY2 = sys.version_info.major == 2
 
 
 ROOT_LOG = logging.getLogger('root')
 
 
+class LocalStoreClient():
+    client = None
+    
+    def __init__(self, redis_db=0, local_store=None):
+        if local_store:
+            self.client = StrictRedis(
+                charset="utf-8",
+                decode_responses=True,
+                db=redis_db,
+                host='localhost',
+                port=6379,
+                socket_timeout=5,
+            )
+        else:
+            # ephemeral storage objects.
```

Comment:
These objects only exist in the index endpoint.  We can't save the non persistent storage in the registry since the index endpoint is a different application as the _indexer_info endpoint.